QSB/QSB.cs

```diff
@@ -47,6 +48,24 @@ private void Start()
             gameObject.AddComponent<GeyserManager>();
             gameObject.AddComponent<OrbManager>();
             gameObject.AddComponent<QSBSectorManager>();
+            gameObject.AddComponent<ConversationManager>();
+
+            Helper.Events.Unity.RunWhen(() => PlayerData.IsLoaded(), RebuildSettingsSave);
+        }
+
+        private void RebuildSettingsSave()
```

Comment:
does this fix a bug?

Reply:
yeah. makes sure time isnt frozen in conversations